packages/ui/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ The following is a curated list of changes in the Enact ui module, newest change
 
 ## [unreleased]
 
+### Fixed
+
+- `ui/Scroller`, `ui/VirtualList`, and `ui/VirtualGridList` to scroll correctly on latest Chrome
```

Comment:
```suggestion
- `ui/Scroller`, `ui/VirtualList`, and `ui/VirtualGridList` to scroll correctly on Chrome 85 or higher in RTL locales
```

Reply:
`ui/Scroller` and `ui/VirtualList` would be fine.

Reply:
I fixed it